modules/serverless-ossm-jwt-new-service.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+// * serverless/networking/serverless-ossm-jwt.adoc
+
+[id="serverless-ossm-jwt-new-service_{context}"]
+= Create a Knative service with sidecar injection
+
+.Procedure
+
+. Copy the following into a YAML file:
++
+
+[source,yaml]
+----
+  apiVersion: serving.knative.dev/v1
+  kind: Service
+  metadata:
+    name: hello-example
+  spec:
+    template:
+      metadata:
+        name: hello-example-1
+        annotations:
+          sidecar.istio.io/inject: "true"
+      spec:
+        containers:
+        - image: docker.io/openshift/hello-openshift
+          name: user-container
+    traffic:
+    - latestRevision: true
+      percent: 100
+----
+
+. Apply the YAML file:
++
+
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+. Create a policy that only allows requests with valid JSON Web Tokens (JWT) in your serverless application namespace:
+
+.. Copy the following into a YAML file:
++
+
+[source,yaml]
+----
+apiVersion: authentication.istio.io/v1alpha1
+kind: Policy
+metadata:
+  name: default
+spec:
+  origins:
+  - jwt:
+      issuer: testing@secure.istio.io
+      jwksUri: "https://raw.githubusercontent.com/istio/istio/release-1.6/security/tools/jwt/samples/jwks.json"
+      triggerRules:
+      - excludedPaths:
+        - prefix: /metrics
+        - prefix: /healthz
+  principalBinding: USE_ORIGIN
+----
+
++
+[IMPORTANT]
+====
+The paths `/metrics` and `/healthz` must be included in `excludedPaths`, as they are accessed from system pods in the `knative-serving` namespace.
+====
+
+.. Apply the YAML file:
```

Comment:
With the inclusion of the `[IMPORTANT]` note, it is resetting the lettering sequence so that you have step 3a followed by step 3a. There is a way to set the value to `b` though the workaround alludes me atm. Check with @jboxman as I think he might recall.

Reply:
For now I've moved the note above the code block and that seems to fix it?